packages/perseus/src/renderer.tsx

```diff
@@ -1421,29 +1416,6 @@ class Renderer
         );
     };
 
-    handleRender: (prevProps: Props) => void = (prevProps: Props) => {
```

Comment:
Tested this earlier with Jeremy and we're fairly certain nothing is calling this. I'm going to be testing this upstream to see if I can notice any regressions.